tests/lvmo/test_lvm_node_reboot_catalogsource.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import tier4a, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM, wait_for_lvm_pod_running
+from ocs_ci.ocs.exceptions import CommandFailed, CatalogSourceNotFoundAfterReboot
+from ocs_ci.ocs.node import wait_for_nodes_status
+from ocs_ci.deployment.deployment import create_catalog_source
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.node import get_node_objs
+
+logger = logging.getLogger(__name__)
+
+
+@tier4a
+@skipif_lvm_not_installed
+@skipif_ocs_version("<4.11")
+class TestLvmCatalogSourceNodeReboot(ManageTest):
+    """
+    Test lvm snapshot bigger than disk
+
+    """
+
+    number_of_reboots = 10
```

Comment:
Just from running into the catalogsource disappearing problem I thought that 10 would catch the problem.
It was so funny, I had the problem and sent Jenkins run with 200 reboots to realize the statistics and it didn't happen :)) Probably it relates to the OCP version that it was fixed and because of that didn't open a BZ.

Reply:
I was wondering, didn't find an option to choose specific OCP image in ocs-ci code. Is there such option beside choosing 4.10, 4.11, ga etc?